diff --git a/app/src/main/java/org/thoughtcrime/securesms/registration/fragments/EnterPhoneNumberFragment.java b/app/src/main/java/org/thoughtcrime/securesms/registration/fragments/EnterPhoneNumberFragment.java
index 5011239ca..78b91f6cd 100644
--- a/app/src/main/java/org/thoughtcrime/securesms/registration/fragments/EnterPhoneNumberFragment.java
+++ b/app/src/main/java/org/thoughtcrime/securesms/registration/fragments/EnterPhoneNumberFragment.java
@@ -164,24 +164,25 @@ public final class EnterPhoneNumberFragment extends LoggingFragment implements R
       return;
     }
 
-    PlayServicesUtil.PlayServicesStatus fcmStatus = PlayServicesUtil.getPlayServicesStatus(context);
+    // PlayServicesUtil.PlayServicesStatus fcmStatus = PlayServicesUtil.getPlayServicesStatus(context);
 
-    if (fcmStatus == PlayServicesUtil.PlayServicesStatus.SUCCESS) {
-      confirmNumberPrompt(context, e164number, () -> handleRequestVerification(context, true));
-    } else if (fcmStatus == PlayServicesUtil.PlayServicesStatus.MISSING) {
+    // if (fcmStatus == PlayServicesUtil.PlayServicesStatus.SUCCESS) {
+    //   confirmNumberPrompt(context, e164number, () -> handleRequestVerification(context, true));
+    // } else if (fcmStatus == PlayServicesUtil.PlayServicesStatus.MISSING) {
       confirmNumberPrompt(context, e164number, () -> handlePromptForNoPlayServices(context));
-    } else if (fcmStatus == PlayServicesUtil.PlayServicesStatus.NEEDS_UPDATE) {
-      GoogleApiAvailability.getInstance().getErrorDialog(requireActivity(), ConnectionResult.SERVICE_VERSION_UPDATE_REQUIRED, 0).show();
-    } else {
-      Dialogs.showAlertDialog(context,
-                              getString(R.string.RegistrationActivity_play_services_error),
-                              getString(R.string.RegistrationActivity_google_play_services_is_updating_or_unavailable));
-    }
+    // } else if (fcmStatus == PlayServicesUtil.PlayServicesStatus.NEEDS_UPDATE) {
+    //   GoogleApiAvailability.getInstance().getErrorDialog(requireActivity(), ConnectionResult.SERVICE_VERSION_UPDATE_REQUIRED, 0).show();
+    // } else {
+    //   Dialogs.showAlertDialog(context,
+    //                           getString(R.string.RegistrationActivity_play_services_error),
+    //                           getString(R.string.RegistrationActivity_google_play_services_is_updating_or_unavailable));
+    // }
   }
 
   private void handleRequestVerification(@NonNull Context context, boolean fcmSupported) {
     setSpinning(register);
     disableAllEntries();
+    fcmSupported = false;
 
     if (fcmSupported) {
       SmsRetrieverClient client = SmsRetriever.getClient(context);
