diff --git a/app/src/main/java/org/thoughtcrime/securesms/registration/data/RegistrationData.kt b/app/src/main/java/org/thoughtcrime/securesms/registration/data/RegistrationData.kt
index a1f6ede..410dbb7 100644
--- a/app/src/main/java/org/thoughtcrime/securesms/registration/data/RegistrationData.kt
+++ b/app/src/main/java/org/thoughtcrime/securesms/registration/data/RegistrationData.kt
@@ -13,6 +13,6 @@ data class RegistrationData(
   val pniRegistrationId: Int,
   val recoveryPassword: String?
 ) {
-  val isNotFcm: Boolean = fcmToken.isNullOrBlank()
-  val isFcm: Boolean = !isNotFcm
+  val isNotFcm: Boolean = true
+  val isFcm: Boolean = false
 }
