diff --git a/app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java b/app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java
index 2ab6bb1..df30f8b 100644
--- a/app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java
+++ b/app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java
@@ -445,30 +445,30 @@ public class ApplicationContext extends Application implements AppForegroundObse
       return;
     }
 
-    PlayServicesUtil.PlayServicesStatus playServicesStatus = PlayServicesUtil.getPlayServicesStatus(this);
-
-    if (playServicesStatus == PlayServicesUtil.PlayServicesStatus.SUCCESS && !SignalStore.account().isFcmEnabled()) {
-      Log.i(TAG, "Play Services are newly-available. Enabling FCM and updating server.");
-      SignalStore.account().setFcmEnabled(true);
-      AppDependencies.getJobManager().startChain(new FcmRefreshJob())
-                                      .then(new RefreshAttributesJob())
-                                      .enqueue();
-    } else if (playServicesStatus == PlayServicesUtil.PlayServicesStatus.MISSING && SignalStore.account().isFcmEnabled()) {
+    // PlayServicesUtil.PlayServicesStatus playServicesStatus = PlayServicesUtil.getPlayServicesStatus(this);
+
+    // if (playServicesStatus == PlayServicesUtil.PlayServicesStatus.SUCCESS && !SignalStore.account().isFcmEnabled()) {
+    //   Log.i(TAG, "Play Services are newly-available. Enabling FCM and updating server.");
+    //   SignalStore.account().setFcmEnabled(true);
+    //   AppDependencies.getJobManager().startChain(new FcmRefreshJob())
+    //                                   .then(new RefreshAttributesJob())
+    //                                   .enqueue();
+    // } else if (playServicesStatus == PlayServicesUtil.PlayServicesStatus.MISSING && SignalStore.account().isFcmEnabled()) {
       Log.w(TAG, "Play Services are no longer available. Disabling FCM and updating server.");
       SignalStore.account().setFcmEnabled(false);
       SignalStore.account().setFcmToken(null);
       AppDependencies.getJobManager().add(new RefreshAttributesJob());
-    } else if (SignalStore.account().isFcmEnabled()) {
-      long lastSetTime = SignalStore.account().getFcmTokenLastSetTime();
-      long nextSetTime = lastSetTime + TimeUnit.HOURS.toMillis(6);
-      long now         = System.currentTimeMillis();
-
-      if (SignalStore.account().getFcmToken() == null || nextSetTime <= now || lastSetTime > now) {
-        AppDependencies.getJobManager().add(new FcmRefreshJob());
-      }
-    } else {
-      Log.d(TAG, "Play Services status: " + playServicesStatus + ", fcmEnabled: false. Skipping FCM check.");
-    }
+    // } else if (SignalStore.account().isFcmEnabled()) {
+    //   long lastSetTime = SignalStore.account().getFcmTokenLastSetTime();
+    //   long nextSetTime = lastSetTime + TimeUnit.HOURS.toMillis(6);
+    //   long now         = System.currentTimeMillis();
+
+    //   if (SignalStore.account().getFcmToken() == null || nextSetTime <= now || lastSetTime > now) {
+    //     AppDependencies.getJobManager().add(new FcmRefreshJob());
+    //   }
+    // } else {
+    //   Log.d(TAG, "Play Services status: " + playServicesStatus + ", fcmEnabled: false. Skipping FCM check.");
+    // }
   }
 
   private void initializeExpiringMessageManager() {
