diff --git a/app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java b/app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java
index ae85e0b..d7d744a 100644
--- a/app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java
+++ b/app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java
@@ -454,35 +454,35 @@ public class ApplicationContext extends Application implements AppForegroundObse
       return;
     }
 
-    PlayServicesUtil.PlayServicesStatus playServicesStatus = PlayServicesUtil.getPlayServicesStatus(this);
-
-    if (playServicesStatus == PlayServicesUtil.PlayServicesStatus.SUCCESS && !SignalStore.account().isFcmEnabled()) {
-      Log.w(TAG, "Play Services are newly-available. Enabling FCM and updating server.");
-      SignalStore.account().setFcmEnabled(true);
-      AppDependencies.getJobManager().startChain(new FcmRefreshJob())
-                                      .then(new RefreshAttributesJob())
-                                      .enqueue();
-      AppDependencies.resetNetwork();
-      AppDependencies.startNetwork();
-      IncomingMessageObserver.stopForegroundService(this);
-    } else if (playServicesStatus == PlayServicesUtil.PlayServicesStatus.MISSING && SignalStore.account().isFcmEnabled()) {
-      Log.w(TAG, "Play Services are no longer available. Attempting to get an FCM token anyway.");
-      AppDependencies.getJobManager().add(new FcmRefreshJob());
-    } else if (playServicesStatus == PlayServicesUtil.PlayServicesStatus.MISSING && (System.currentTimeMillis() - SignalStore.misc().getLastMissingPlayServicesFcmVerificationTime()) > TimeUnit.DAYS.toMillis(3)) {
-      Log.i(TAG, "Play Services are unavailable, but it's been long enough that we should check and see if we can get an FCM token anyway.");
-      AppDependencies.getJobManager().add(new FcmRefreshJob());
-    } else if (SignalStore.account().isFcmEnabled()) {
-      long lastSetTime = SignalStore.account().getFcmTokenLastSetTime();
-      long nextSetTime = lastSetTime + TimeUnit.HOURS.toMillis(6);
-      long now         = System.currentTimeMillis();
-
-      if (SignalStore.account().getFcmToken() == null || nextSetTime <= now || lastSetTime > now) {
-        Log.i(TAG, "Time for routine FCM token refresh.");
-        AppDependencies.getJobManager().add(new FcmRefreshJob());
-      }
-    } else {
+    // PlayServicesUtil.PlayServicesStatus playServicesStatus = PlayServicesUtil.getPlayServicesStatus(this);
+
+    // if (playServicesStatus == PlayServicesUtil.PlayServicesStatus.SUCCESS && !SignalStore.account().isFcmEnabled()) {
+    //   Log.w(TAG, "Play Services are newly-available. Enabling FCM and updating server.");
+    //   SignalStore.account().setFcmEnabled(true);
+    //   AppDependencies.getJobManager().startChain(new FcmRefreshJob())
+    //                                   .then(new RefreshAttributesJob())
+    //                                   .enqueue();
+    //   AppDependencies.resetNetwork();
+    //   AppDependencies.startNetwork();
+    //   IncomingMessageObserver.stopForegroundService(this);
+    // } else if (playServicesStatus == PlayServicesUtil.PlayServicesStatus.MISSING && SignalStore.account().isFcmEnabled()) {
+    //   Log.w(TAG, "Play Services are no longer available. Attempting to get an FCM token anyway.");
+    //   AppDependencies.getJobManager().add(new FcmRefreshJob());
+    // } else if (playServicesStatus == PlayServicesUtil.PlayServicesStatus.MISSING && (System.currentTimeMillis() - SignalStore.misc().getLastMissingPlayServicesFcmVerificationTime()) > TimeUnit.DAYS.toMillis(3)) {
+    //   Log.i(TAG, "Play Services are unavailable, but it's been long enough that we should check and see if we can get an FCM token anyway.");
+    //   AppDependencies.getJobManager().add(new FcmRefreshJob());
+    // } else if (SignalStore.account().isFcmEnabled()) {
+    //   long lastSetTime = SignalStore.account().getFcmTokenLastSetTime();
+    //   long nextSetTime = lastSetTime + TimeUnit.HOURS.toMillis(6);
+    //   long now         = System.currentTimeMillis();
+
+    //   if (SignalStore.account().getFcmToken() == null || nextSetTime <= now || lastSetTime > now) {
+    //     Log.i(TAG, "Time for routine FCM token refresh.");
+    //     AppDependencies.getJobManager().add(new FcmRefreshJob());
+    //   }
+    // } else {
       Log.d(TAG, "Play Services status: " + playServicesStatus + ", fcmEnabled: false. Skipping FCM check.");
-    }
+    // }
   }
 
   private void initializeExpiringMessageManager() {
