diff --git a/app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java b/app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java
index 9efd93d..9c1fd55 100644
--- a/app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java
+++ b/app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java
@@ -376,13 +376,13 @@ public class ApplicationContext extends MultiDexApplication implements AppForegr
   }
 
   private void initializeFcmCheck() {
-    if (SignalStore.account().isRegistered()) {
-      long nextSetTime = SignalStore.account().getFcmTokenLastSetTime() + TimeUnit.HOURS.toMillis(6);
+    // if (SignalStore.account().isRegistered()) {
+    //   long nextSetTime = SignalStore.account().getFcmTokenLastSetTime() + TimeUnit.HOURS.toMillis(6);
 
-      if (SignalStore.account().getFcmToken() == null || nextSetTime <= System.currentTimeMillis()) {
-        ApplicationDependencies.getJobManager().add(new FcmRefreshJob());
-      }
-    }
+    //   if (SignalStore.account().getFcmToken() == null || nextSetTime <= System.currentTimeMillis()) {
+    //     ApplicationDependencies.getJobManager().add(new FcmRefreshJob());
+    //   }
+    // }
   }
 
   private void initializeExpiringMessageManager() {
