diff --git a/app/src/main/java/org/thoughtcrime/securesms/jobs/FcmRefreshJob.java b/app/src/main/java/org/thoughtcrime/securesms/jobs/FcmRefreshJob.java
index 8af65ea7..dc2ed835 100644
--- a/app/src/main/java/org/thoughtcrime/securesms/jobs/FcmRefreshJob.java
+++ b/app/src/main/java/org/thoughtcrime/securesms/jobs/FcmRefreshJob.java
@@ -84,30 +84,30 @@ public class FcmRefreshJob extends BaseJob {
     if (!SignalStore.account().isFcmEnabled()) return;
 
     Log.i(TAG, "Reregistering FCM...");
-
-    int result = GoogleApiAvailability.getInstance().isGooglePlayServicesAvailable(context);
-
-    if (result != ConnectionResult.SUCCESS) {
-      notifyFcmFailure();
-    } else {
-      Optional<String> token = FcmUtil.getToken(context);
-
-      if (token.isPresent()) {
-        String oldToken = SignalStore.account().getFcmToken();
-
-        if (!token.get().equals(oldToken)) {
-          int oldLength = oldToken != null ? oldToken.length() : -1;
-          Log.i(TAG, "Token changed. oldLength: " + oldLength + "  newLength: " + token.get().length());
-        } else {
-          Log.i(TAG, "Token didn't change.");
-        }
-
-        NetworkResultUtil.toBasicLegacy(SignalNetwork.account().setFcmToken(token.get()));
-        SignalStore.account().setFcmToken(token.get());
-      } else {
-        throw new RetryLaterException(new IOException("Failed to retrieve a token."));
-      }
-    }
+    return;
+    // int result = GoogleApiAvailability.getInstance().isGooglePlayServicesAvailable(context);
+
+    // if (result != ConnectionResult.SUCCESS) {
+    //   notifyFcmFailure();
+    // } else {
+    //   Optional<String> token = FcmUtil.getToken(context);
+
+    //   if (token.isPresent()) {
+    //     String oldToken = SignalStore.account().getFcmToken();
+
+    //     if (!token.get().equals(oldToken)) {
+    //       int oldLength = oldToken != null ? oldToken.length() : -1;
+    //       Log.i(TAG, "Token changed. oldLength: " + oldLength + "  newLength: " + token.get().length());
+    //     } else {
+    //       Log.i(TAG, "Token didn't change.");
+    //     }
+
+    //     NetworkResultUtil.toBasicLegacy(SignalNetwork.account().setFcmToken(token.get()));
+    //     SignalStore.account().setFcmToken(token.get());
+    //   } else {
+    //     throw new RetryLaterException(new IOException("Failed to retrieve a token."));
+    //   }
+    // }
   }
 
   @Override
