diff --git a/app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java b/app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java
index 7b69285..a2cf42c 100644
--- a/app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java
+++ b/app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java
@@ -229,7 +229,7 @@ public class ApplicationContext extends MultiDexApplication implements AppForegr
     ApplicationDependencies.getMegaphoneRepository().onAppForegrounded();
     ApplicationDependencies.getDeadlockDetector().start();
     SubscriptionKeepAliveJob.enqueueAndTrackTimeIfNecessary();
-    FcmFetchManager.onForeground(this);
+    //FcmFetchManager.onForeground(this);
 
     SignalExecutors.BOUNDED.execute(() -> {
       FeatureFlags.refreshIfNecessary();
@@ -368,10 +368,10 @@ public class ApplicationContext extends MultiDexApplication implements AppForegr
 
   private void initializeFcmCheck() {
     if (SignalStore.account().isRegistered()) {
-      long nextSetTime = SignalStore.account().getFcmTokenLastSetTime() + TimeUnit.HOURS.toMillis(6);
-
-      if (SignalStore.account().getFcmToken() == null || nextSetTime <= System.currentTimeMillis()) {
-        ApplicationDependencies.getJobManager().add(new FcmRefreshJob());
+//      long nextSetTime = SignalStore.account().getFcmTokenLastSetTime() + TimeUnit.HOURS.toMillis(6);
+//
+//      if (SignalStore.account().getFcmToken() == null || nextSetTime <= System.currentTimeMillis()) {
+//        ApplicationDependencies.getJobManager().add(new FcmRefreshJob());
       }
     }
   }

