diff --git a/app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java b/app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java
index 5240edd..88561dc 100644
--- a/app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java
+++ b/app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java
@@ -347,7 +347,7 @@ public class ApplicationContext extends MultiDexApplication implements AppForegr
       long nextSetTime = SignalStore.account().getFcmTokenLastSetTime() + TimeUnit.HOURS.toMillis(6);
 
       if (SignalStore.account().getFcmToken() == null || nextSetTime <= System.currentTimeMillis()) {
-        ApplicationDependencies.getJobManager().add(new FcmRefreshJob());
+        //ApplicationDependencies.getJobManager().add(new FcmRefreshJob());
       }
     }
   }
@@ -412,7 +412,7 @@ public class ApplicationContext extends MultiDexApplication implements AppForegr
     if (TextSecurePreferences.getNeedsMessagePull(this)) {
       Log.i(TAG, "Scheduling a message fetch.");
       if (Build.VERSION.SDK_INT >= 26) {
-        FcmJobService.schedule(this);
+        // FcmJobService.schedule(this);
       } else {
         ApplicationDependencies.getJobManager().add(new PushNotificationReceiveJob());
       }

