diff --git a/app/src/main/java/org/thoughtcrime/securesms/registration/v2/ui/phonenumber/EnterPhoneNumberV2Fragment.kt b/app/src/main/java/org/thoughtcrime/securesms/registration/v2/ui/phonenumber/EnterPhoneNumberV2Fragment.kt
index 78c1f33..60ae619 100644
--- a/app/src/main/java/org/thoughtcrime/securesms/registration/v2/ui/phonenumber/EnterPhoneNumberV2Fragment.kt
+++ b/app/src/main/java/org/thoughtcrime/securesms/registration/v2/ui/phonenumber/EnterPhoneNumberV2Fragment.kt
@@ -432,34 +432,35 @@ class EnterPhoneNumberV2Fragment : LoggingFragment(R.layout.fragment_registratio
   }
 
   private fun validateFcmStatus(context: Context): Boolean {
-    val fcmStatus = PlayServicesUtil.getPlayServicesStatus(context)
-    Log.d(TAG, "Got $fcmStatus for Play Services status.")
-    when (fcmStatus) {
-      PlayServicesUtil.PlayServicesStatus.SUCCESS -> {
-        return true
-      }
-
-      PlayServicesUtil.PlayServicesStatus.MISSING -> {
-        fragmentViewModel.setError(EnterPhoneNumberV2State.Error.PLAY_SERVICES_MISSING)
-        return false
-      }
-
-      PlayServicesUtil.PlayServicesStatus.NEEDS_UPDATE -> {
-        fragmentViewModel.setError(EnterPhoneNumberV2State.Error.PLAY_SERVICES_NEEDS_UPDATE)
-        return false
-      }
-
-      PlayServicesUtil.PlayServicesStatus.TRANSIENT_ERROR -> {
-        fragmentViewModel.setError(EnterPhoneNumberV2State.Error.PLAY_SERVICES_TRANSIENT)
-        return false
-      }
-
-      null -> {
-        Log.w(TAG, "Null result received from PlayServicesUtil, marking Play Services as missing.")
-        fragmentViewModel.setError(EnterPhoneNumberV2State.Error.PLAY_SERVICES_MISSING)
-        return false
-      }
-    }
+    return false
+    // val fcmStatus = PlayServicesUtil.getPlayServicesStatus(context)
+    // Log.d(TAG, "Got $fcmStatus for Play Services status.")
+    // when (fcmStatus) {
+    //   PlayServicesUtil.PlayServicesStatus.SUCCESS -> {
+    //     return true
+    //   }
+
+    //   PlayServicesUtil.PlayServicesStatus.MISSING -> {
+    //     fragmentViewModel.setError(EnterPhoneNumberV2State.Error.PLAY_SERVICES_MISSING)
+    //     return false
+    //   }
+
+    //   PlayServicesUtil.PlayServicesStatus.NEEDS_UPDATE -> {
+    //     fragmentViewModel.setError(EnterPhoneNumberV2State.Error.PLAY_SERVICES_NEEDS_UPDATE)
+    //     return false
+    //   }
+
+    //   PlayServicesUtil.PlayServicesStatus.TRANSIENT_ERROR -> {
+    //     fragmentViewModel.setError(EnterPhoneNumberV2State.Error.PLAY_SERVICES_TRANSIENT)
+    //     return false
+    //   }
+
+    //   null -> {
+    //     Log.w(TAG, "Null result received from PlayServicesUtil, marking Play Services as missing.")
+    //     fragmentViewModel.setError(EnterPhoneNumberV2State.Error.PLAY_SERVICES_MISSING)
+    //     return false
+    //   }
+    // }
   }
 
   private fun handleConfirmNumberDialogCanceled() {
