diff --git a/app/src/main/java/org/thoughtcrime/securesms/jobs/FcmRefreshJob.java b/app/src/main/java/org/thoughtcrime/securesms/jobs/FcmRefreshJob.java
index 84c712b..06ed1bd 100644
--- a/app/src/main/java/org/thoughtcrime/securesms/jobs/FcmRefreshJob.java
+++ b/app/src/main/java/org/thoughtcrime/securesms/jobs/FcmRefreshJob.java
@@ -70,33 +70,34 @@ public class FcmRefreshJob extends BaseJob {
   @Override
   public void onRun() throws Exception {
     if (!SignalStore.account().isFcmEnabled()) return;
+    return;
 
-    Log.i(TAG, "Reregistering FCM...");
+    // Log.i(TAG, "Reregistering FCM...");
 
-    int result = GoogleApiAvailability.getInstance().isGooglePlayServicesAvailable(context);
+    // int result = GoogleApiAvailability.getInstance().isGooglePlayServicesAvailable(context);
 
-    if (result != ConnectionResult.SUCCESS) {
-      Log.w(TAG, "Play Services are unavailable. Skipping FCM refresh.");
-      return;
-    }
+    // if (result != ConnectionResult.SUCCESS) {
+    //   Log.w(TAG, "Play Services are unavailable. Skipping FCM refresh.");
+    //   return;
+    // }
 
-    Optional<String> token = FcmUtil.getToken(context);
+    // Optional<String> token = FcmUtil.getToken(context);
 
-    if (token.isPresent()) {
-      String oldToken = SignalStore.account().getFcmToken();
+    // if (token.isPresent()) {
+    //   String oldToken = SignalStore.account().getFcmToken();
 
-      if (!token.get().equals(oldToken)) {
-        int oldLength = oldToken != null ? oldToken.length() : -1;
-        Log.i(TAG, "Token changed. oldLength: " + oldLength + "  newLength: " + token.get().length());
-      } else {
-        Log.i(TAG, "Token didn't change.");
-      }
+    //   if (!token.get().equals(oldToken)) {
+    //     int oldLength = oldToken != null ? oldToken.length() : -1;
+    //     Log.i(TAG, "Token changed. oldLength: " + oldLength + "  newLength: " + token.get().length());
+    //   } else {
+    //     Log.i(TAG, "Token didn't change.");
+    //   }
 
-      NetworkResultUtil.toBasicLegacy(SignalNetwork.account().setFcmToken(token.get()));
-      SignalStore.account().setFcmToken(token.get());
-    } else {
-      throw new RetryLaterException(new IOException("Failed to retrieve a token."));
-    }
+    //   NetworkResultUtil.toBasicLegacy(SignalNetwork.account().setFcmToken(token.get()));
+    //   SignalStore.account().setFcmToken(token.get());
+    // } else {
+    //   throw new RetryLaterException(new IOException("Failed to retrieve a token."));
+    // }
   }
 
   @Override
