diff --git a/app/src/main/java/org/thoughtcrime/securesms/logsubmit/LogSectionSystemInfo.java b/app/src/main/java/org/thoughtcrime/securesms/logsubmit/LogSectionSystemInfo.java
index 3831e29..7564c89 100644
--- a/app/src/main/java/org/thoughtcrime/securesms/logsubmit/LogSectionSystemInfo.java
+++ b/app/src/main/java/org/thoughtcrime/securesms/logsubmit/LogSectionSystemInfo.java
@@ -176,8 +176,9 @@ public class LogSectionSystemInfo implements LogSection {
   }
 
   private static String getPlayServicesString(@NonNull Context context) {
-    int result = GoogleApiAvailability.getInstance().isGooglePlayServicesAvailable(context);
-    return result == ConnectionResult.SUCCESS ? "true" : "false (" + result + ")";
+//    int result = GoogleApiAvailability.getInstance().isGooglePlayServicesAvailable(context);
+//    return result == ConnectionResult.SUCCESS ? "true" : "false (" + result + ")";
+    return "false()";
   }
 
   private static String getEmojiVersionString(@NonNull Context context) {
