diff --git a/app/src/main/java/org/thoughtcrime/securesms/jobs/FcmRefreshJob.java b/app/src/main/java/org/thoughtcrime/securesms/jobs/FcmRefreshJob.java
index c809e22..ef66861 100644
--- a/app/src/main/java/org/thoughtcrime/securesms/jobs/FcmRefreshJob.java
+++ b/app/src/main/java/org/thoughtcrime/securesms/jobs/FcmRefreshJob.java
@@ -79,44 +79,44 @@ public class FcmRefreshJob extends BaseJob {
       Log.w(TAG, "Play Services are unavailable.");
     }
 
-    Optional<String> token = FcmUtil.getToken(context);
+    // Optional<String> token = FcmUtil.getToken(context);
 
-    if (token.isPresent()) {
-      if (playServicesMissing) {
-        Log.w(TAG, "We were able to get a token despite Play Services being missing!");
-      }
+    // if (token.isPresent()) {
+    //   if (playServicesMissing) {
+    //     Log.w(TAG, "We were able to get a token despite Play Services being missing!");
+    //   }
       
-      String oldToken = SignalStore.account().getFcmToken();
-
-      if (!token.get().equals(oldToken)) {
-        int oldLength = oldToken != null ? oldToken.length() : -1;
-        Log.i(TAG, "Token changed. oldLength: " + oldLength + "  newLength: " + token.get().length());
-      } else {
-        Log.i(TAG, "Token didn't change.");
-      }
-
-      NetworkResultUtil.toBasicLegacy(SignalNetwork.account().setFcmToken(token.get()));
-      SignalStore.account().setFcmToken(token.get());
-
-      if (!SignalStore.account().isFcmEnabled()) {
-        Log.w(TAG, "We had no Play Services, but were still able to get an FCM token! Re-enabling.");
-        SignalStore.account().setFcmEnabled(true);
-        AppDependencies.getJobManager().add(new RefreshAttributesJob());
-        AppDependencies.resetNetwork();
-        AppDependencies.startNetwork();
-        IncomingMessageObserver.stopForegroundService(context);
-      }
-
-      if (SignalStore.settings().getForceWebsocketMode() == ForceWebsocketMode.ENABLED_AUTOMATICALLY) {
-        Log.i(TAG, "FCM succeeded while in auto-enabled websocket mode. Reverting to disabled.");
-        SignalStore.settings().setForceWebsocketMode(ForceWebsocketMode.DISABLED);
-        IncomingMessageObserver.stopForegroundService(context);
-        AppDependencies.resetNetwork();
-        AppDependencies.startNetwork();
-      }
-    } else {
-      throw new RetryLaterException(new IOException("Failed to retrieve a token."));
-    }
+    //   String oldToken = SignalStore.account().getFcmToken();
+
+    //   if (!token.get().equals(oldToken)) {
+    //     int oldLength = oldToken != null ? oldToken.length() : -1;
+    //     Log.i(TAG, "Token changed. oldLength: " + oldLength + "  newLength: " + token.get().length());
+    //   } else {
+    //     Log.i(TAG, "Token didn't change.");
+    //   }
+
+    //   NetworkResultUtil.toBasicLegacy(SignalNetwork.account().setFcmToken(token.get()));
+    //   SignalStore.account().setFcmToken(token.get());
+
+    //   if (!SignalStore.account().isFcmEnabled()) {
+    //     Log.w(TAG, "We had no Play Services, but were still able to get an FCM token! Re-enabling.");
+    //     SignalStore.account().setFcmEnabled(true);
+    //     AppDependencies.getJobManager().add(new RefreshAttributesJob());
+    //     AppDependencies.resetNetwork();
+    //     AppDependencies.startNetwork();
+    //     IncomingMessageObserver.stopForegroundService(context);
+    //   }
+
+    //   if (SignalStore.settings().getForceWebsocketMode() == ForceWebsocketMode.ENABLED_AUTOMATICALLY) {
+    //     Log.i(TAG, "FCM succeeded while in auto-enabled websocket mode. Reverting to disabled.");
+    //     SignalStore.settings().setForceWebsocketMode(ForceWebsocketMode.DISABLED);
+    //     IncomingMessageObserver.stopForegroundService(context);
+    //     AppDependencies.resetNetwork();
+    //     AppDependencies.startNetwork();
+    //   }
+    // } else {
+    //   throw new RetryLaterException(new IOException("Failed to retrieve a token."));
+    // }
   }
 
   @Override
