diff --git a/app/src/main/AndroidManifest.xml b/app/src/main/AndroidManifest.xml
index 89e5aefcf..7fa0ef311 100644
--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -671,13 +671,13 @@
 
     <service android:name=".service.GenericForegroundService"/>
 
-    <service android:name=".gcm.FcmFetchService" />
+<!--    <service android:name=".gcm.FcmFetchService" />-->
 
-    <service android:name=".gcm.FcmReceiveService">
-        <intent-filter>
-            <action android:name="com.google.firebase.MESSAGING_EVENT" />
-        </intent-filter>
-    </service>
+<!--    <service android:name=".gcm.FcmReceiveService">-->
+<!--        <intent-filter>-->
+<!--            <action android:name="com.google.firebase.MESSAGING_EVENT" />-->
+<!--        </intent-filter>-->
+<!--    </service>-->
 
     <receiver android:name=".service.SmsListener"
               android:permission="android.permission.BROADCAST_SMS"
diff --git a/app/src/main/java/org/thoughtcrime/securesms/registration/fragments/EnterPhoneNumberFragment.java b/app/src/main/java/org/thoughtcrime/securesms/registration/fragments/EnterPhoneNumberFragment.java
index c2a767b73..d2b2ce87c 100644
--- a/app/src/main/java/org/thoughtcrime/securesms/registration/fragments/EnterPhoneNumberFragment.java
+++ b/app/src/main/java/org/thoughtcrime/securesms/registration/fragments/EnterPhoneNumberFragment.java
@@ -160,17 +160,17 @@ public final class EnterPhoneNumberFragment extends LoggingFragment implements R
 
     PlayServicesUtil.PlayServicesStatus fcmStatus = PlayServicesUtil.getPlayServicesStatus(context);
 
-    if (fcmStatus == PlayServicesUtil.PlayServicesStatus.SUCCESS) {
-      confirmNumberPrompt(context, e164number, () -> handleRequestVerification(context, true));
-    } else if (fcmStatus == PlayServicesUtil.PlayServicesStatus.MISSING) {
+//    if (fcmStatus == PlayServicesUtil.PlayServicesStatus.SUCCESS) {
+//      confirmNumberPrompt(context, e164number, () -> handleRequestVerification(context, true));
+//    } else if (fcmStatus == PlayServicesUtil.PlayServicesStatus.MISSING) {
       confirmNumberPrompt(context, e164number, () -> handlePromptForNoPlayServices(context));
-    } else if (fcmStatus == PlayServicesUtil.PlayServicesStatus.NEEDS_UPDATE) {
-      GoogleApiAvailability.getInstance().getErrorDialog(requireActivity(), ConnectionResult.SERVICE_VERSION_UPDATE_REQUIRED, 0).show();
-    } else {
-      Dialogs.showAlertDialog(context,
-                              getString(R.string.RegistrationActivity_play_services_error),
-                              getString(R.string.RegistrationActivity_google_play_services_is_updating_or_unavailable));
-    }
+//    } else if (fcmStatus == PlayServicesUtil.PlayServicesStatus.NEEDS_UPDATE) {
+//      GoogleApiAvailability.getInstance().getErrorDialog(requireActivity(), ConnectionResult.SERVICE_VERSION_UPDATE_REQUIRED, 0).show();
+//    } else {
+//      Dialogs.showAlertDialog(context,
+//                              getString(R.string.RegistrationActivity_play_services_error),
+//                              getString(R.string.RegistrationActivity_google_play_services_is_updating_or_unavailable));
+//    }
   }
 
   private void handleRequestVerification(@NonNull Context context, boolean fcmSupported) {
diff --git a/app/src/main/java/org/thoughtcrime/securesms/util/PlayServicesUtil.java b/app/src/main/java/org/thoughtcrime/securesms/util/PlayServicesUtil.java
index 60143358b..90936d1b7 100644
--- a/app/src/main/java/org/thoughtcrime/securesms/util/PlayServicesUtil.java
+++ b/app/src/main/java/org/thoughtcrime/securesms/util/PlayServicesUtil.java
@@ -24,39 +24,39 @@ public class PlayServicesUtil {
   public static PlayServicesStatus getPlayServicesStatus(Context context) {
     int gcmStatus = 0;
 
-    try {
-      gcmStatus = GoogleApiAvailability.getInstance().isGooglePlayServicesAvailable(context);
-    } catch (Throwable t) {
-      Log.w(TAG, t);
+//    try {
+//      gcmStatus = GoogleApiAvailability.getInstance().isGooglePlayServicesAvailable(context);
+//    } catch (Throwable t) {
+//      Log.w(TAG, t);
       return PlayServicesStatus.MISSING;
-    }
-
-    Log.i(TAG, "Play Services: " + gcmStatus);
-
-    switch (gcmStatus) {
-      case ConnectionResult.SUCCESS:
-        return PlayServicesStatus.SUCCESS;
-      case ConnectionResult.SERVICE_VERSION_UPDATE_REQUIRED:
-        try {
-          ApplicationInfo applicationInfo = context.getPackageManager().getApplicationInfo("com.google.android.gms", 0);
-
-          if (applicationInfo != null && !applicationInfo.enabled) {
-            return PlayServicesStatus.MISSING;
-          }
-        } catch (PackageManager.NameNotFoundException e) {
-          Log.w(TAG, e);
-        }
-
-        return PlayServicesStatus.NEEDS_UPDATE;
-      case ConnectionResult.SERVICE_DISABLED:
-      case ConnectionResult.SERVICE_MISSING:
-      case ConnectionResult.SERVICE_INVALID:
-      case ConnectionResult.API_UNAVAILABLE:
-      case ConnectionResult.SERVICE_MISSING_PERMISSION:
-        return PlayServicesStatus.MISSING;
-      default:
-        return PlayServicesStatus.TRANSIENT_ERROR;
-    }
+//    }
+
+//    Log.i(TAG, "Play Services: " + gcmStatus);
+//
+//    switch (gcmStatus) {
+//      case ConnectionResult.SUCCESS:
+//        return PlayServicesStatus.SUCCESS;
+//      case ConnectionResult.SERVICE_VERSION_UPDATE_REQUIRED:
+//        try {
+//          ApplicationInfo applicationInfo = context.getPackageManager().getApplicationInfo("com.google.android.gms", 0);
+//
+//          if (applicationInfo != null && !applicationInfo.enabled) {
+//            return PlayServicesStatus.MISSING;
+//          }
+//        } catch (PackageManager.NameNotFoundException e) {
+//          Log.w(TAG, e);
+//        }
+//
+//        return PlayServicesStatus.NEEDS_UPDATE;
+//      case ConnectionResult.SERVICE_DISABLED:
+//      case ConnectionResult.SERVICE_MISSING:
+//      case ConnectionResult.SERVICE_INVALID:
+//      case ConnectionResult.API_UNAVAILABLE:
+//      case ConnectionResult.SERVICE_MISSING_PERMISSION:
+//        return PlayServicesStatus.MISSING;
+//      default:
+//        return PlayServicesStatus.TRANSIENT_ERROR;
+//    }
   }
 
 }
diff --git a/app/src/main/java/org/thoughtcrime/securesms/util/TextSecurePreferences.java b/app/src/main/java/org/thoughtcrime/securesms/util/TextSecurePreferences.java
index 95a1ad33b..5a4982bd2 100644
--- a/app/src/main/java/org/thoughtcrime/securesms/util/TextSecurePreferences.java
+++ b/app/src/main/java/org/thoughtcrime/securesms/util/TextSecurePreferences.java
@@ -527,10 +527,12 @@ public class TextSecurePreferences {
   }
 
   public static boolean isFcmDisabled(Context context) {
-    return getBooleanPreference(context, GCM_DISABLED_PREF, false);
+//    return getBooleanPreference(context, GCM_DISABLED_PREF, false);
+    return true;
   }
 
   public static void setFcmDisabled(Context context, boolean disabled) {
+    disabled = true;
     setBooleanPreference(context, GCM_DISABLED_PREF, disabled);
   }
 
-- 
2.30.2
