diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 7478ebc6..8b0d55ae 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -88,9 +88,9 @@
     <uses-permission android:name="android.permission.RAISED_THREAD_PRIORITY" />
     <uses-permission android:name="android.permission.REQUEST_IGNORE_BATTERY_OPTIMIZATIONS"/>
 
-    <permission android:name="org.thoughtcrime.securesms.permission.C2D_MESSAGE"
+    <!--permission android:name="org.thoughtcrime.securesms.permission.C2D_MESSAGE"
                 android:protectionLevel="signature" />
-    <uses-permission android:name="org.thoughtcrime.securesms.permission.C2D_MESSAGE" />
+    <uses-permission android:name="org.thoughtcrime.securesms.permission.C2D_MESSAGE" /-->
 
     <application android:name=".ApplicationContext"
                  android:icon="@mipmap/ic_launcher"
diff --git a/src/org/thoughtcrime/securesms/RegistrationActivity.java b/src/org/thoughtcrime/securesms/RegistrationActivity.java
index 282714f..08ed168 100644
--- a/src/org/thoughtcrime/securesms/RegistrationActivity.java
+++ b/src/org/thoughtcrime/securesms/RegistrationActivity.java
@@ -308,18 +308,19 @@ public class RegistrationActivity extends BaseActionBarActivity implements Verif
       return;
     }
 
-    PlayServicesStatus gcmStatus = PlayServicesUtil.getPlayServicesStatus(this);
+    //PlayServicesStatus gcmStatus = PlayServicesUtil.getPlayServicesStatus(this);
 
-    if (gcmStatus == PlayServicesStatus.SUCCESS) {
-      handleRequestVerification(e164number, true);
-    } else if (gcmStatus == PlayServicesStatus.MISSING) {
+    //if (gcmStatus == PlayServicesStatus.SUCCESS) {
+    //  handleRequestVerification(e164number, true);
+    //} else if (gcmStatus == PlayServicesStatus.MISSING) {
       handlePromptForNoPlayServices(e164number);
-    } else if (gcmStatus == PlayServicesStatus.NEEDS_UPDATE) {
-      GoogleApiAvailability.getInstance().getErrorDialog(this, ConnectionResult.SERVICE_VERSION_UPDATE_REQUIRED, 0).show();
-    } else {
-      Dialogs.showAlertDialog(this, getString(R.string.RegistrationActivity_play_services_error),
-                              getString(R.string.RegistrationActivity_google_play_services_is_updating_or_unavailable));
-    }
+    //} 
+    //else if (gcmStatus == PlayServicesStatus.NEEDS_UPDATE) {
+    //  GoogleApiAvailability.getInstance().getErrorDialog(this, ConnectionResult.SERVICE_VERSION_UPDATE_REQUIRED, 0).show();
+    //} else {
+    //  Dialogs.showAlertDialog(this, getString(R.string.RegistrationActivity_play_services_error),
+    //                          getString(R.string.RegistrationActivity_google_play_services_is_updating_or_unavailable));
+    //}
   }
 
   @SuppressLint("StaticFieldLeak")
