diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 6ae85a346..3bbcf44db 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -473,11 +473,11 @@
 
     <service android:name=".service.GenericForegroundService"/>
 
-    <service android:name=".gcm.FcmService">
-        <intent-filter>
-            <action android:name="com.google.firebase.MESSAGING_EVENT" />
-        </intent-filter>
-    </service>
+    <!--<service android:name=".gcm.FcmService">-->
+        <!--<intent-filter>-->
+            <!--<action android:name="com.google.firebase.MESSAGING_EVENT" />-->
+        <!--</intent-filter>-->
+    <!--</service>-->
 
     <receiver android:name=".service.SmsListener"
               android:permission="android.permission.BROADCAST_SMS"
diff --git a/src/org/thoughtcrime/securesms/RegistrationActivity.java b/src/org/thoughtcrime/securesms/RegistrationActivity.java
index d951fb8b6..42855cf58 100644
--- a/src/org/thoughtcrime/securesms/RegistrationActivity.java
+++ b/src/org/thoughtcrime/securesms/RegistrationActivity.java
@@ -460,18 +460,7 @@ public class RegistrationActivity extends BaseActionBarActivity implements Verif
       return;
     }
 
-    PlayServicesStatus gcmStatus = PlayServicesUtil.getPlayServicesStatus(this);
-
-    if (gcmStatus == PlayServicesStatus.SUCCESS) {
-      handleRequestVerification(e164number, true);
-    } else if (gcmStatus == PlayServicesStatus.MISSING) {
-      handlePromptForNoPlayServices(e164number);
-    } else if (gcmStatus == PlayServicesStatus.NEEDS_UPDATE) {
-      GoogleApiAvailability.getInstance().getErrorDialog(this, ConnectionResult.SERVICE_VERSION_UPDATE_REQUIRED, 0).show();
-    } else {
-      Dialogs.showAlertDialog(this, getString(R.string.RegistrationActivity_play_services_error),
-                              getString(R.string.RegistrationActivity_google_play_services_is_updating_or_unavailable));
-    }
+    handlePromptForNoPlayServices(e164number);
   }
 
   private void handleRequestVerification(@NonNull String e164number, boolean gcmSupported) {
