diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 796b4ef6..05f58b14 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -84,8 +84,8 @@
     <uses-permission android:name="android.permission.WRITE_CALL_LOG"/>
     <uses-permission android:name="android.permission.REQUEST_IGNORE_BATTERY_OPTIMIZATIONS"/>

-    <permission android:name="org.thoughtcrime.securesms.permission.C2D_MESSAGE"
-                android:protectionLevel="signature" />
+<!--    <permission android:name="org.thoughtcrime.securesms.permission.C2D_MESSAGE"
+                android:protectionLevel="signature" />-->
     <uses-permission android:name="org.thoughtcrime.securesms.permission.C2D_MESSAGE" />

     <application android:name=".ApplicationContext"
diff --git a/src/org/thoughtcrime/securesms/RegistrationActivity.java b/src/org/thoughtcrime/securesms/RegistrationActivity.java
index b30bafee..3cbd777f 100644
--- a/src/org/thoughtcrime/securesms/RegistrationActivity.java
+++ b/src/org/thoughtcrime/securesms/RegistrationActivity.java
@@ -210,16 +210,16 @@ public class RegistrationActivity extends BaseActionBarActivity {

       PlayServicesStatus gcmStatus = checkPlayServices(self);

-      if (gcmStatus == PlayServicesStatus.SUCCESS) {
-        promptForRegistrationStart(self, e164number, true);
-      } else if (gcmStatus == PlayServicesStatus.MISSING) {
+      //if (gcmStatus == PlayServicesStatus.SUCCESS) {
+      //  promptForRegistrationStart(self, e164number, true);
+      //} else if (gcmStatus == PlayServicesStatus.MISSING) {
         promptForNoPlayServices(self, e164number);
-      } else if (gcmStatus == PlayServicesStatus.NEEDS_UPDATE) {
-        GoogleApiAvailability.getInstance().getErrorDialog(self, ConnectionResult.SERVICE_VERSION_UPDATE_REQUIRED, 0).show();
-      } else {
-        Dialogs.showAlertDialog(self, getString(R.string.RegistrationActivity_play_services_error),
-                                getString(R.string.RegistrationActivity_google_play_services_is_updating_or_unavailable));
-      }
+      //} else if (gcmStatus == PlayServicesStatus.NEEDS_UPDATE) {
+      //  GoogleApiAvailability.getInstance().getErrorDialog(self, ConnectionResult.SERVICE_VERSION_UPDATE_REQUIRED, 0).show();
+      //} else {
+      //  Dialogs.showAlertDialog(self, getString(R.string.RegistrationActivity_play_services_error),
+      //                          getString(R.string.RegistrationActivity_google_play_services_is_updating_or_unavailable));
+      //}
     }

     private void promptForRegistrationStart(final Context context, final String e164number, final boolean gcmSupported) {